### ISSUE TITLE
🐛 BUG: Trailing commas in parentheses get removed from markdown

### ISSUE THREAD

**Issue by tusharsadhwani** [snowpackjs/astro#1525]
### What package manager are you using?

npm

### What operating system are you using?

Linux

### Describe the Bug

If you have this code:

````astro
<Markdown>
  ```python
  (a,)
  ```
</Markdown>
````

It gets rid of the trailing comma. Output:

```python
(a)
```

### Steps to Reproduce

Codesandbox below :)

### Link to Minimal Reproducible Example (Optional)

https://codesandbox.io/s/pensive-goldstine-gz6ry?file=/src/pages/index.astro

**Comment by matthewp** [comment]
Trying to verify this is fixed, blocked by https://github.com/snowpackjs/astro/issues/1735

**Comment by matthewp** [comment]
Ok, I can verify this will be fixed in `next`! https://stackblitz.com/edit/github-jhuzfh?file=src/pages/index.astro

**Comment by tusharsadhwani** [comment]
Wait, isn't next already merged? Not released I suppose?

**Comment by natemoo-re** [comment]
@tusharsadhwani Yes, `next` is merged to `main`. It's released under `astro@next`, but will be upgraded to `astro` as soon as possible.

The team has decided to close out issues that have been confirmed as fixed by [astro@0.21.0-next.0](https://github.com/snowpackjs/astro/releases/tag/astro%400.21.0-next.0), [astro@0.21.0-next.1](https://github.com/snowpackjs/astro/releases/tag/astro%400.21.0-next.1), or [astro@0.21.0-next.2](https://github.com/snowpackjs/astro/releases/tag/astro%400.21.0-next.2). Our hope is that this will help the [v0.21 milestone](https://github.com/snowpackjs/astro/milestone/1) remain as actionable as possible.

To verify that this issue has been fixed, you may
- Open a new [`astro@next` project on StackBlitz](https://astro.new/minimal@next)
- Locally upgrade to the `astro@next` preview
    ```shell
    npm i astro@next
    # If using Framework renderers...
    npm i @astrojs/renderer-name@next
    ```

If you think we made a mistake, **please reopen this issue**!

If you run into any other problems with `astro@next`, please [open a new issue](https://github.com/snowpackjs/astro/issues/new?assignees=&labels=0.21&template=---01-bug-report.yml&title=%F0%9F%90%9B+BUG%3Aastro@next).
